google/ggrc-core
# GGRC-7054 Improvements for Task Group in Global Search

# Issue description

Task group in global search have some view issues.

# Steps to test the changes

1. Log in ggrc-app and open Global Search.
2. Select Task Group object type and click on Search.
3. By clicking grey triangle expand additional info.
Expected: Task Group popup doesn't contain Notes.
4. Add 'Details' to Task Group and open this Task Group in Global Search.
Expected: 'Details' field should be displayed for Task Group.
5. Look at the layout of columns.
Expected: Values in Columns should be aligned.
6. Click on 'Open in a new tab' link.
Expected: Link should follow on Setup tab on Workflow page.


# Solution description

Added additional condition for rendering of Task Group. Added class for fixing margin-right.

# Sanity checklist




# PR Review checklist
